forest/shared/src/cli/mod.rs

```diff
@@ -30,6 +30,66 @@ pub static FOREST_VERSION_STRING: Lazy<String> =
 /// CLI options
 #[derive(StructOpt, Debug)]
 pub struct CliOpts {
+    /// A TOML file containing relevant configurations
+    #[structopt(short, long)]
+    pub config: Option<String>,
+    /// Client JWT token to use for JSON-RPC authentication
+    #[structopt(short, long)]
+    pub token: Option<String>,
+    /// Address used for RPC. By defaults binds on localhost on port 1234.
+    #[structopt(long)]
+    pub rpc_address: Option<SocketAddr>,
+    /// Address used for metrics collection server. By defaults binds on localhost on port 6116.
+    #[structopt(long)]
+    pub metrics_address: Option<SocketAddr>,
```

Comment:
I don't think we use it anywhere in the `forest-cli` unless I'm missing something.
```suggestion
```